app/models/concerns/court/postcode_courts.rb

```diff
@@ -28,8 +28,9 @@ def self.postcode_court_matcher(postcode)
 
       def ingest_new_postcode_courts(postcodes)
         postcodes.split(",").map do |postcode|
-          postcode = postcode.gsub(/[^0-9a-z ]/i, "").downcase
+          postcode = UKPostcode.parse(postcode).to_s
```

Comment:
@leoapost yes it's a gem for validating a format of a postcode. I'm using it for formatting and to avoid SQL queries if not necessary.